app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,114 @@
+    ) do
+      extend(Forwardable)
+      def_delegators(:@authenticator_status_input, :authenticator_name, :account,
+                     :username, :webservice, :status_webservice)
+
+      def call
+        validate_generic_status_validations
+        create_authentication_parameters
+        validate_secrets
+      end
+
+      private
+
+      def validate_generic_status_validations
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatingJwtStatusConfiguration.new)
+        validate_account_exists
+        validate_user_has_access_to_status_webservice
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedUserHasAccessToStatusWebservice.new)
+        validate_authenticator_webservice_exists
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedAuthenticatorWebServiceExists.new)
+        validate_webservice_is_whitelisted
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedStatusWebserviceIsWhitelisted.new)
+        validate_service_id_exists
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedStatusServiceIdExists.new)
+      end
+
+      def validate_user_has_access_to_status_webservice
+        @validate_role_can_access_webservice.(
+          webservice: status_webservice,
+            account: account,
+            user_id: username,
+            privilege: 'read'
+        )
+      end
+
+      def validate_webservice_is_whitelisted
+        @validate_webservice_is_whitelisted.(
+          webservice: webservice,
+            account: account,
+            enabled_authenticators: @enabled_authenticators
+        )
+      end
+
+      def validate_authenticator_webservice_exists
+        @validate_webservice_exists.(
+          webservice: webservice,
+            account: account
+        )
+      end
+
+      def validate_service_id_exists
+        raise Errors::Authentication::AuthnJwt::ServiceIdMissing unless @authenticator_status_input.service_id
+      end
+
+      def validate_account_exists
+        @validate_account_exists.(
+          account: @authenticator_status_input.account
+        )
+      end
+
+      def validate_secrets
+        validate_signing_key_secrets
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedSigningKeyConfiguration.new)
+        validate_issuer
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedIssuerConfiguration.new)
+        validate_identity_secrets
+        @logger.debug(LogMessages::Authentication::AuthnJwt::ValidatedIdentityConfiguration.new)
+      end
+
+      def validate_signing_key_secrets
+        @create_signing_key.call(authentication_parameters: @authentication_parameters)
+      end
+
+      def validate_issuer
+        @fetch_issuer_value.call(authentication_parameters: @authentication_parameters)
+      end
+
+      def validate_identity_secrets
+        @fetch_identity_from_token.new(@authentication_parameters).identity_configured_properly?
```

Comment:
According to this code if the customer didnt configured the `token-app-property` variable the status check will failed, but the authenticator can work with the user in the url request instead of inside the token.

So in this check you are enforcing the customer to 1 configuration of user inside the token